site/en/posts/chromeos-112-113-release-notes.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'ChromeOS 112 and 113 improve Screen Capture, screensavers, and security'
```

Comment:
All of the other titles are "ChromeOS X release notes"; can we normalize the title?